presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -109,6 +109,24 @@ Name                                   Description
 ``KerberosKeytabPath``                 Kerberos keytab file.
 ``KerberosCredentialCachePath``        Kerberos credential cache.
 ``extraCredentials``                   Extra credentials for connecting to external services. The
-                                       extraCredentials is a list of key-value pairs. Example:
+                                       ``extraCredentials`` is a list of key-value pairs. Example:
                                        ``foo:bar;abc:xyz`` will create credentials ``abc=xyz`` and ``foo=bar``
+``roles``                              Authorization roles that will be set for catalogs. The
```

Comment:
A list of key-value pairs defining catalog and authorization role to use for the catalog. For example,  the configuration``catalog1:roleA;catalog2:roleB`` causes Presto to use ``roleA`` to connect to ``catalog1`` and ``roleB`` to connect to ``catalog2``.

Reply:
I like the original phrasing better, since it first tells you what it is, rather than starting off with the exact format.
```
Authorization roles to use for catalogs, specified as a list of key-value pairs for the
catalog and role. For example, ``catalog1:roleA;catalog2:roleB`` will set ``roleA``
for ``catalog1`` and ``roleB`` for ``catalog2``.

Reply:
Lets not use future tense at least please.